### PR TITLE
Props for paths keys

### PR DESCRIPTION
**Pull request by alisaono**
Added new props to Map:
'pathIDKey': String, defaults to "id". Key for the area IDs of SVG paths.
'pathTitleKey': String, defaults to "title". Key for the area titles of SVG paths.

#17 @sanjaypojo @AlmahaAlmalki

### CODE REVIEW

**Comment by alisaono** [comment]
This might actually result in a merge conflict with the other branch #19 
You might not want to merge this when you approve... I can merge it locally and push once you approve this pull request.
@sanjaypojo

**Review by sanjaypojo — APPROVED** [review]
Looks great @alisaono 👍

**Pull request merged**
